webknossos-datastore/app/com/scalableminds/webknossos/datastore/DataStoreConfig.scala

```diff
@@ -7,41 +7,40 @@ import play.api.Configuration
 import scala.concurrent.duration._
 
 class DataStoreConfig @Inject()(configuration: Configuration) extends ConfigReader {
-  override def raw = configuration
+  override def raw: Configuration = configuration
 
   object Http {
-    val uri = get[String]("http.uri")
+    val uri: String = get[String]("http.uri")
   }
 
   object Braingames {
     object Binary {
       object ChangeHandler {
-        val enabled = get[Boolean]("braingames.binary.changeHandler.enabled")
-        val tickerInterval = get[Int]("braingames.binary.changeHandler.tickerInterval") minutes
+        val enabled: Boolean = get[Boolean]("braingames.binary.changeHandler.enabled")
+        val tickerInterval: FiniteDuration = get[Int]("braingames.binary.changeHandler.tickerInterval") minutes
```

Comment:
Why would this not be inferred automatically? Just curious :cat2:

Reply:
You’re right, it is inferred automatically, which is why the previous version compiled too. However, Scala style recommendation is to explicitly type-annotate all public members, both for readability and for compiler speed-up.